tensorflow/compiler/xla/service/gpu/miopen_conv_algorithm_picker.cc

```diff
@@ -32,9 +32,8 @@ namespace {
 
 using absl::optional;
 using se::DeviceMemoryBase;
-using se::dnn::AlgorithmConfig;
 using se::dnn::AlgorithmDesc;
-
+  
```

Comment:
nit: clang-format?

Reply:
yes...forgot to do that...will do it before the actual merge

Reply:
done